src/js/main.js

```diff
@@ -119,10 +122,41 @@ function doFaction(faction, teamMemberElements) {
                     table.appendChild(tr);
                 });
 
+                const divContainer = el('div');
+                setStyle(divContainer, {
+                    borderTop: '1px solid rgb(31, 31, 31)',
+                    cursor: 'pointer',
+                    height: statsExpanded ? '100%' : '26px'
+                });
+                const chevron = divContainer.appendChild(el('span'));
```

Comment:
This function was quite huge already. Can we extract building chevron to another helper fucntion at least?